### PR TITLE
YLP-2125 Allow users to ask support to resend the verification email

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/pages/login/verify-email-page.tsx [hunk 1]
```diff
@@ -25,67 +25,172 @@
...
 const VerifyEmailPage: FunctionComponent = () => {
-  const { loginWithRedirect, logout } = useAuth0()
+  const { classes: { appBar, desktopLogo } } = classes()
+  const { loginWithRedirect, getAccessTokenSilently } = useAuth0()
   const { t } = useTranslation()
-  const theme = useTheme()
+  const { logout: authLogout } = useAuth()
+  const alert = useAlert()
+
+  const sendConfirmationEmail = async (): Promise<void> => {
```
Comment (fatb38) [inline packages/yourloops/pages/login/verify-email-page.tsx:66]: I suggest a more approriate function name that precise we are contacting the support (and not sending a confirmation email). Maybe `contactCustomerSupport` ?

File: cloudfront-dist/deployment/docker.template.env [hunk 2]
```diff
@@ -23,6 +23,7 @@ ASSETS_URL=publicAssetBuketUrl
 METRICS_SERVICE=disabled
 STONLY_WID=disabled
 SUPPORT_WEB_ADDRESS=https://yourloops.zendesk.com/hc/requests/new
+CONTACT_SUPPORT_WEB_URL=https://yourloops.zendesk.com/hc/requests/new
```
Comment (fatb38) [inline cloudfront-dist/deployment/docker.template.env:26]: **❔ question:** ‏I don't figure out why there is here a new env var which is the same as `SUPPORT_WEB_ADDRESS` ? Couldn't you reuse it ?

File: packages/yourloops/test/integration/pages/verify-email/verify-email-page.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    expect(pageContent.getByText('Verify your email address')).toBeVisible()
+    expect(pageContent.getByText('To use YourLoops, you need to confirm your email address by clicking on the link we sent you. This helps to keep your account secure.')).toBeVisible()
+
+    const contactSupportSection = pageContent.getByTestId('verify-email-details-2')
+    expect(contactSupportSection).toHaveTextContent('No mail in your inbox or spam folder? Contact the customer support.')
+    expect(within(contactSupportSection).getByRole('button', { name: 'Contact the customer support' })).toBeVisible()
+
+    const createNewAccountSection = pageContent.getByTestId('verify-email-details-3')
+    expect(createNewAccountSection).toHaveTextContent('Wrong address? If you mistyped your email when signing up, create a new account.')
+    expect(within(createNewAccountSection).getByRole('button', { name: 'create a new account' })).toBeVisible()
```
Comment (fatb38) [inline packages/yourloops/test/integration/pages/verify-email/verify-email-page.spec.tsx:69]: **🎯 suggestion (non-blocking):** ‏Links click could be tested too 😉
```suggestion
    const createNewAccountButton = within(createNewAccountSection).getByRole('button', { name: 'create a new account' })
    expect(createNewAccountButton).toBeVisible()
    await userEvent.click(createNewAccountButton)
    expect(logoutMock).toHaveBeenCalledTimes(1)
    expect(loginWithRedirectMock).toHaveBeenCalledTimes(1)
```

**Review by TimCanu — APPROVED** [review]

**Pull request merged**